src/FSharp.Control.Websockets.TPL/FSharp.Control.Websockets.TPL.fs

```diff
@@ -66,6 +66,11 @@ module WebSocket =
     open System.Net.WebSockets
     open FSharp.Control.Tasks.V2
 
+    #if NETSTANDARD2_1
```

Comment:
Should these checks be the other way around so that when NETSTANDARD2_2 or even further beyond happens they don't have to be revisited?